reaktive/src/commonMain/kotlin/com/badoo/reaktive/completable/AndThen.kt

```diff
@@ -0,0 +1,31 @@
+package com.badoo.reaktive.completable
+
+import com.badoo.reaktive.base.ErrorCallback
+import com.badoo.reaktive.base.Observer
+import com.badoo.reaktive.base.subscribeSafe
+import com.badoo.reaktive.disposable.Disposable
+import com.badoo.reaktive.disposable.DisposableWrapper
+
+fun Completable.andThen(completable: Completable): Completable =
+    completableUnsafe { observer ->
+        val disposableWrapper = DisposableWrapper()
+        observer.onSubscribe(disposableWrapper)
+
+        subscribeSafe(
+            object : CompletableObserver, Observer, ErrorCallback by observer {
```

Comment:
`Observer` is useless and can be omitted, `CompletableObserver` interface extends `Observer`